app/src/main/java/org/fossasia/susi/ai/device/deviceconnect/adapters/recycleradapters/RoomsAdapter.kt

```diff
@@ -34,6 +34,7 @@ class RoomsAdapter(private val availableRoomsList: ArrayList<DeviceConnectFragme
                 val roomNameClicked = availableRoomsList[adapterPosition].room
                 var intent = Intent(context, DeviceActivity::class.java)
                 intent.putExtra("roomName", roomNameClicked)
+                intent.putExtra("connectTo", "DeviceConnectFragment")
```

Comment:
Why is it used?

Reply:
The DeviceActivity is the base activity for 2 major features. The intent parameter is passed to tell the activity which fragment it should load.

Reply:
Make it a constant

Reply:
Updated